src/config.h

```diff
@@ -138,6 +138,7 @@ namespace config {
     } dd;
 
     int min_fps_factor;  // Minimum fps target, determines minimum frame time
+    int max_bitrate; // Maximum bitrate, sets ceiling in kbps for bitrate requested from client
```

Comment:
One lint error to resolve and I think this will be good to merge. https://github.com/LizardByte/Sunshine/actions/runs/13226866711/job/36919249645?pr=3628#step:7:26

Reply:
done.